core/modules/wdb/ChunkResource.h

```diff
@@ -191,8 +189,7 @@ class ChunkResourceMgr {
     using Ptr = std::shared_ptr<ChunkResourceMgr>;
 
     /// Factory
-    static Ptr newMgr(mysql::MySqlConfig const& c);
-    static Ptr newFakeMgr();
+    static Ptr newMgr(SQLBackend::Ptr const& backend);
```

Comment:
I see it. By the way, why did you remove "newFakeMgr"? Was it meant for the unit-testing of the class?